content/docs/reference-react.md

```diff
@@ -332,7 +332,11 @@ Reține că pentru randarea componentelor `lazy` este necesară randarea unui co
 
 ### `React.Suspense` {#reactsuspense}
 
+<<<<<<< HEAD
 `React.Suspense` iți permite să specifici indicatorul de încărcare în cazul în care unele componente din ierarhia copil nu sunt gata pentru randare. În prezent, componentele lazy loading sunt **singurele** cazuri suportate de `<React.Suspense>`:
+=======
+`React.Suspense` lets you specify the loading indicator in case some components in the tree below it are not yet ready to render. Today, lazy loading components is the **only** use case supported by `<React.Suspense>`:
+>>>>>>> 081bb31226919062938ef924472ba1b4170facfc
```

Comment:
This conflict needs to be resolved! The Romanian translation is not affected by the upstream change

Reply:
@tesseralis I would like to resolve this conflict and merge the PR. It's simple since the Romanian translation is not affected by the upstream change. Unfortunately even if you added me as a maintainer I don't have permissions to modify the file and merge the PR. Is there something I need to do for this to happen?